enhancements/authentication/oauth-resource-storage.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Secure OAuth Resource Storage
+authors:
+  - "@sttts"
+reviewers:
+  - "@mfojtik"
+  - "@deads2k"
+  - "@stlaz"
+  - "@marun"
+  - "@polynomial"
+approvers:
+  - "@mfoktik"
+  - "anand"
+creation-date: 2020-03-30
+last-updated: 2020-05-12
+status: provisional
+see-also:
+  - "/enhancements/kube-apiserver/encrypting-data-at-datastore-layer.md"  
+replaces:
+superseded-by:
+---
+
+# Secure OAuth Resource Storage
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions [optional]
+
+1. Do we really need token read access for users. This enhancement allows us to add that with reasonable effort,
```

Comment:
Assign this as a PM question.  There may be a user requirement to see and expire "assigned" tokens or tokens that have been named or annotated in some way with an intent.  Similar to the legacy app token integration with github or google

Reply:
@mcurry-rh @Anandnatraj

Reply:
From Slack: a simple logout-all-other-sessions endpoints would be enough. We will probably go that route. Though I have remove all the details around this topic from the enhancement for now.

Reply:
Moved to a follow-up. Not part of this release.